templates/bridgeserver2.yaml

```diff
@@ -492,6 +543,7 @@ Resources:
         config:
           ignore_checks:
             - E3012
+    Condition: CreateR53Record
```

Comment:
should the `CreateDevResources` condition include uat now?  or maybe there should be a separate `CreateUatResources` condition?

Reply:
CreateDevResources is sort of misnamed. It's really more of a hack to create Local Resources, and we piggyback it off of the Dev environment. We wouldn't want to add this to Staging because we don't want to create Local Resources twice.

I ended up creating the CreateR53Record condition because (a) it was easier than figuring out of boolean logic could be used with multiple conditions and (b) I wanted to specifically disable Route53 in Dev AWS. Given that the sagebridge.org domain lives in the Prod AWS account, it should probably live in the AWS account configuration, not the Bridge stack configuration. But I didn't want to pull in too many changes at once and make the AWS Migration even more complicated than it already is.